templates/case/summary.html.twig

```diff
@@ -2,7 +2,8 @@
 
 {% trans_default_domain 'case' %}
 
-{% set createdate = case.createdAt|date(format_date) %}
+{% set createdate = case.receivedAt|date(format_date) %}
```

Comment:
Consider renaming to `receivedate`? Or maybe even `received_date`?

Reply:
Updated to `received_date` to follow twig naming convention.